File: practicas/unidad2/python/Persona.py
class Persona:
    def __init__(self, nombre, edad):
        self.nombre = nombre
        self.edad = edad
        self.vive = True

    # Se usa la palabra self para hacer referencia al objeto actual
    def es_mayor_que(self, otra_persona):
        return self.edad > otra_persona.edad

File: practicas/unidad2/ruby/Persona.rb
class Persona
    # otro código
  
    # se usa la palabra self para hacer referencia al objeto actual
    def es_mayor_que(otra_persona)
      return self.edad > otra_persona.edad
    end
  
    # es posible acceder a los atributos utilizando un @ antes del nombre del atributo
    def es_mayor_de_edad
      @edad > 18 # la palabra return es opcional
    end
  
end